doc/examples/cloud-config-user-groups.txt

```diff
@@ -20,14 +20,14 @@ users:
     groups: users
     selinux_user: staff_u
     expiredate: '2032-09-01'
-    ssh_import_id: foobar
+    ssh_import_id: <Launchpad/Github id>
```

Comment:
These examples are incorrect as ssh_import_id needs to be a list.

Since these are two examples, maybe we could make them explicit for either gh: or lp:?
```suggestion
    ssh_import_id: [lp:falcojr, gh:TheRealFalcon]
```